src/main/java/org/opentripplanner/routing/algorithm/AStar.java

```diff
@@ -180,10 +184,10 @@ boolean iterate(){
 
 //                LOG.info("{} {}", v, remaining_w);
 
-                if (remaining_w < 0 || Double.isInfinite(remaining_w) ) {
+                if (remaining_w < 0 || Double.isInfinite(remaining_w)) {
                     continue;
                 }
-                double estimate = v.getWeight() + remaining_w;
+                double estimate = v.getWeight() + remaining_w * runState.options.remainingWeightWeight;
```

Comment:
To jest jedyna faktycznie zmieniona linia. Chciałem pominąć reformat kodu ale niestety IDE zrobiło go automatycznie podczas commita